### ISSUE TITLE
REUSE compliance

### ISSUE THREAD

**Issue by lmoureaux** [longturn/freeciv21#809]
**Is your feature request related to a problem? Please describe.**
A nice standard solving our problems with copyright headers and licenses: https://reuse.software/tutorial/

**Describe the solution you'd like**
Implement REUSE (maybe)

**Describe alternatives you've considered**
Do nothing

**Additional context**
Need to keep track of the licenses used for distribution

**Comment by lmoureaux** [comment]
REUSE isn't implemented --> reopening

**Comment by jwrober** [comment]
What needs to occur to call this "implemented"? I figured documenting it and using it was implemented.

**Comment by lmoureaux** [comment]
> What needs to occur to call this "implemented"? I figured documenting it and using it was implemented.

Running `reuse lint` shouldn't produce significant output.
Maybe by closing this you meant that using REUSE was accepted as a good practice?

**Comment by jwrober** [comment]
> Maybe by closing this you meant that using REUSE was accepted as a good practice?

Yes and documenting the usage for "go forward" changes means we adopt and use it. `reuse lint` will get smaller as time goes by.
